content/millennium/r4/clinical/care-provision/care-team.md

```diff
@@ -54,34 +54,34 @@ _Implementation Notes_
  `_id`        | Required if `patient` or `encounter` is not present | [`token`]     | The logical resource id associated with the resource.
  `patient`    | Required if `_id` or `encounter` is not present     | [`reference`] | Who care team is for. Example: `patient=12345`
  `encounter`  | Required if `_id` or `patient` is not present       | [`reference`] | Who care team is for. Example: `encounter=98765`
- `category`   | N                                                   | [`token`]     | The scope of care team being searched for. Examples: `category=longitudinal`
+ `category`   | N                                                   | [`token`]     | The scope of care team being searched for. Example: `category=longitudinal`
  `status`     | N                                                   | [`token`]     | Indicates the status of the care team
-`_revinclude`      | No                                             | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example:_revinclude=Provenance:target
+`_revinclude` | N                                                   | [`token`]     | Provenance resource entries to be returned as part of the bundle. Example: `_revinclude=Provenance:target`
 
 Notes:
 
-* The `_id` parameter
-  * This is required if `patient` or `encounter` are not provided.
+- The `_id` parameter
+  - This is required if `patient` or `encounter` are not provided.
 
-* The `patient` parameter
-  * This is required when `_id` or `encounter` is not provided.
-  * This can be combined with `status` as per US Core guidelines.
-  * This can be combined with the `category` parameter.
+- The `patient` parameter
+  - This is required when `_id` or `encounter` is not provided.
+  - This can be combined with `status` as per US Core guidelines.
+  - This can be combined with the `category` parameter.
 
-* The `encounter` parameter
-  * This is required when `_id` or `patient` is not provided.
-  * This cannot be combined with `category` or status.
+- The `encounter` parameter
+  - This is required when `_id` or `patient` is not provided.
```

Comment:
"are not provided"

Reply:
> "are not provided"

please fix this to get this merged

Reply:
@gmodrogan  - pushing change now.  
You should be able to merge shortly

![image](https://github.com/cerner/fhir.cerner.com/assets/144141792/f426d24c-b6e9-4819-8ec5-065dfda71e23)